src/assets/asset-transfers.md

```diff
@@ -220,8 +220,9 @@ curl -X GET "https://service.centrapay.com/api/asset-transfers?senderAccountId=a
       "description": "$60 Giftcard",
       "message": "Happy birthday",
       "senderName": "My Cafe",
-      "recipientId": "49105001-384a-40ed-b666-3d56dd1aa2c5",
```

Comment:
Just blocking this till we hear back from our integrators on how they're using it.